File: 4 - Estrutura de Repetição/Exercicio.13.py
#13. Desenvolva um programa que permite para calcular a soma de números a partir do número 1 
# até um número informado pelo usuário. Por exemplo, se o usuário informar o número 5, o
# programa deverá calcular: 1 + 2 + 3 + 4 + 5 = 15, e mostrar o resultado para o usuário.

numero = -2
while numero<0: 
    numero=int(input("Digite um número: "))
    if numero<0: print("digite um numero maior qure 0")
    else :
     soma=0
for c in range(1,numero+1):
 soma += c
print(f'A soma dos números de 1 até {numero} é igual a {soma}.')

File: Livro - Módulo Java/4 - Estrutura de Repetição/Exercicio13.java
import java.util.Scanner;

public class Exercicio13 {
    public static void main(String[] args) {
        // Desenvolva um programa que permite calcular a soma de números a partir do número 1 até um número informado pelo usuário.

        Scanner scanner = new Scanner(System.in);

        System.out.print("Digite um número para calcular a soma até esse número: ");
        int numeroUsuario = scanner.nextInt();

        if (numeroUsuario > 0) {
            int resultadoSoma = calcularSomaAteN(numeroUsuario);

            System.out.println("Resultado");
            System.out.println("A soma de 1 até " + numeroUsuario + " é " + resultadoSoma);
        } else {
            // Exibir uma mensagem de erro se o número não for válido
            System.out.println("Erro");
            System.out.println("Por favor, digite um número válido maior que zero");
        }
    }

    public static int calcularSomaAteN(int numero) {
        int soma = 0;

        for (int i = 1; i <= numero; i++) {
            soma += i;
        }

        return soma;
    }
}